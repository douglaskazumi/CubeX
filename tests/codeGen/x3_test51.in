"Characters in the core language which have no obvious ASCII equivalent are represented in the full language in ASCII as follows:"
"Expressions (denoted e in the grammar) may be surrounded by parentheses."