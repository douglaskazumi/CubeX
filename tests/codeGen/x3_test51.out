
36, 37, 38, 39, 40, 36, 37, 38, 39, 40, 36, 37, 38, 39, 40, 
Characters in the core language which have no obvious ASCII equivalent are represented in the full language in ASCII as follows:Characters in the core language which have no obvious ASCII equivalent are represented in the full language in ASCII as follows:Characters in the core language which have no obvious ASCII equivalent are represented in the full language in ASCII as follows:Characters in the core language which have no obvious ASCII equivalent are represented in the full language in ASCII as follows:Characters in the core language which have no obvious ASCII equivalent are represented in the full language in ASCII as follows:Characters in the core language which have no obvious ASCII equivalent are represented in the full language in ASCII as follows:Characters in the core language which have no obvious ASCII equivalent are represented in the full language in ASCII as follows:Characters in the core language which have no obvious ASCII equivalent are represented in the full language in ASCII as follows:Characters in the core language which have no obvious ASCII equivalent are represented in the full language in ASCII as follows:Characters in the core language which have no obvious ASCII equivalent are represented in the full language in ASCII as follows:
Expressions (denoted e in the grammar) may be surrounded by parentheses.Expressions (denoted e in the grammar) may be surrounded by parentheses.Expressions (denoted e in the grammar) may be surrounded by parentheses.Expressions (denoted e in the grammar) may be surrounded by parentheses.Expressions (denoted e in the grammar) may be surrounded by parentheses.Expressions (denoted e in the grammar) may be surrounded by parentheses.Expressions (denoted e in the grammar) may be surrounded by parentheses.Expressions (denoted e in the grammar) may be surrounded by parentheses.Expressions (denoted e in the grammar) may be surrounded by parentheses.Expressions (denoted e in the grammar) may be surrounded by parentheses.
